pytest_plugins/markers.py

```diff
@@ -14,6 +14,11 @@ def pytest_configure(config):
         "upgrade: Upgrade tests",
         "run_in_one_thread: Sequential tests",
         "build_sanity: Fast, basic tests that confirm build is ready for full test suite",
+        "cloud_provisioning: Tests that runs on cloud Providers",
+        "on_premises_provisioning: Tests that runs on on_premises Providers",
+        "libvirt_discovery: Tests depends on Libvirt Provider for discovery",
+        "libvirt_content_host: Tests depends on Libvirt Provider for content hosts",
+        "auth_source: Tests depends on external auth sources for login to satellite"
```

Comment:
I think this marker (and maybe the others eventually) should be applied to the fixtures for auth source entity creation.

This way the test author doesn't have to add the fixture, and apply marks when they've used the fixture.

Reply:
@mshriver Cool ! Thanks will do !

Reply:
Done !

Reply:
@mshriver I noticed that markers assigned to fixtures are not playing any role is test selection/deselection atleast in collect only :(

Do you feel otherwise ?

Reply:
@jyejare we would have to use a plugin like https://pypi.org/project/pytest-fixture-marker/ or pytest hooks to enforce this, out of the scope of your PR here.